### ISSUE TITLE
PHP conf dir

### ISSUE THREAD

**Issue by nueko** [lukesampson/scoop#1401]
Hi, there is something weird when i created `scoop/apps/php/conf` directory on Windows and run `scoop status` command.
```bash
> scoop status
Scoop is up to date.
These apps failed to install:
    php
```

**Comment by lukesampson** [comment]
Scoop sees everything under `~/scoop/apps/php` as a version of PHP, but if it can't find install info inside the directory it thinks the install failed.

Did you mean to create the directory at `~/scoop/apps/php/current/conf` instead? That should work, assuming you have installed PHP first and you can see `~/scoop/apps/php/current` already exists.

**Comment by nueko** [comment]
Isn't creating `conf` inside the `current` folder means that we should move the `conf` folder on every update?

**Comment by rrelmy** [comment]
This is a duplicate of #1185 and will be fixed with #1372
We will put the configuration under `~/scoop/data/php/conf/`

You could temporary change the environment variable `PHP_INI_SCAN_DIR`.
